packages/shared/types/workspace.ts

```diff
@@ -4,4 +4,22 @@ export type Workspace = {
   sortBy: SortBy;
   projects: Project[];
   missing: string[];
+  templates: Template[];
+};
+
+export type Template = {
+  title: string;
+  tags: string[] | null;
+  github_link: string;
+  description: string;
+  image_1: string | null;
+  play_link: string | null;
+  id: number;
+  video_1: string | null;
```

Comment:
are the properties `image_1` and `video_1` named with the `_1` on purpose or was this a typo?

Reply:
they are named like that on the `https://studios.decentraland.org/api/get/resources` response